prac_05/state_names.py

```diff
@@ -0,0 +1,25 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TO DO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory",
+                "WA": "Western Australia", "ACT": "Australian Capital Territory",
+                "VIC": "Victoria", "TAS": "Tasmania"}
+MENU = """Type the state code to reveal its name
+To list all codes and names press enter anytime."""
```

Comment:
Is the program supposed to end when they list all names? There's no loop to return to input when nothing is entered.